website/docs/plugins/ax_sweeper.md

```diff
@@ -35,29 +35,29 @@ The return value of the function should be the value that we want to optimize.
 To compute the best parameters for the Banana function, clone the code and run the following command in the `plugins/hydra_ax_sweeper` directory:
 
 ```
-python example/banana.py -m 'banana.x=int(interval(-5, 5))' 'banana.y=interval(-5, 10.1)'
+python example/banana.py -m 'banana.x=int(interval(-5, 5))' 'banana.y=interval(-5, 10.1)' 'hydra.sweeper.ax_config.is_errorless=True'
```

Comment:
1. no need to quote regular overrides (you need to quote the functions because the shell will not like ().
2. use lowercase true.